Automattic/wp-calypso
# Remove content block

#### Proposed Changes

Remove content block sources since the block will live in Blocks Everywhere plugin.
As a side effect, it will also stop registering content block which should help with copy-pasting issues.

#### Testing Instructions

* Run yarn dev --sync in apps/happy-blocks to sync code to your sandbox
* Create a new topic
* Verify Upgrade block still works. Verification is required b/c Upgrade block shares some sources with Content block. 

#### Pre-merge Checklist





Related to #
